### ISSUE TITLE
Discussion about this project/sample

### ISSUE THREAD

**Issue by zabojad** [elsassph/modular-haxe-example#2]
@elsassph What about packaging Require.hx and Stub.hx in a little haxelib that could be improved over time by the community?

Also what about, in the meantime, starting a discussion with the Haxe team to start thinking about the addition of similar features natively in the Haxe compiler?

**Comment by elsassph** [comment]
Yes, I've improved/cloned it in the last 2 projects I shared. It may be time to package it.
